pages/news-and-events/community-spotlight/success-stories/_id.vue

```diff
@@ -295,25 +301,22 @@ export default {
 
 .seperator-path {
   width: 100%;
-  height: 2px;
+  height: 0.125rem;
   background: rgb(216, 216, 216);
   border-radius: 0px;
-  margin-top: 32px;
-  margin-bottom: 32px;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
 }
 
 .story-bold-field {
-  color: rgb(0, 0, 0);
-  font-family: Asap;
   font-size: 14px;
   font-weight: 600;
   text-transform: uppercase;
 }
 
 .story-field {
-  font-family: Asap;
   font-size: 14px;
-  line-height: 24px;
+  line-height: 1.5rem;
 }
```

Comment:
Is there not something in the design system for this? If not there should be.

Reply:
`body4` is 14px but it is the wrong colour to match the wireframes:
https://nih-sparc.github.io/sparc-design-system-components/?path=/story/components-typography--body

Reply:
I would use that and let Gazelle sort it out, the wireframes should be built upon the design system.